dfusion_rust_core/Cargo.toml

```diff
@@ -11,7 +11,7 @@ graph = { git = "https://github.com/graphprotocol/graph-node", tag = "v0.17.0" }
 graph-node-reader = { git = "https://github.com/gnosis/graph-node-reader", tag = "v0.17.0" }
 lazy_static = "1.4.0"
 log = "0.4.8"
-mock-it = "0.3.0"
+mockall = "0.6.0"
```

Comment:
In the mockall repository they recommend to have it as a dev dependency only:

```toml
[dev-dependencies]
mockall = "0.6.0"
```

Can we do this?

Reply:
The `#[automock]` macro is also compiled in non-dev mode and causes compilation errors if mockall is inside dev dependencies. I wasn't yet able to conditionally compile the mock macro for the test config.

Do you have an idea how to do it?

Reply:
Ok, found the explanation: https://github.com/asomers/mockall/issues/77

Reply:
Maybe

```
#[cfg_attr(test, automock)]
trait MyTrait {
    fn foo(&self, x: u32) -> u32;
}
```

or the Readme is not updated, because I noticed that the example uses it as a normal dependency, too.